### PR TITLE
Little style fixes

### PR DESCRIPTION
**Pull request by rdbartlett**
* link group logo to group on thread page
* tidy up markdown styles: list + blockquote + headings
* fix margins on thread items
* remove broken group logo from membership request notification
* make all the icons in the user dropdown line up

**Before:**

image

**After:**

image

---

**Before**

image

**After**

image

---

**Before**

image

**After**

image

---

**Before**

image

**After**
image

### CODE REVIEW

**Comment by gdpelican** [comment]
Hooray!

File: lineman/app/components/group_page/group_theme/group_theme.haml [hunk 1]
```diff
@@ -5,7 +5,8 @@
 
   %section.group-theme__header--compact{ng-if: 'compact', aria-label: "{{ 'thread_group.aria_label | translate'}}"}
     .group-theme__logo--compact{aria-hidden: 'true'}
-      %img{ng-src: "{{group.logoUrl()}}"}
+      %a{lmo-href-for: 'group'}
```
Comment (robguthrie) [inline lineman/app/components/group_page/group_theme/group_theme.haml]: hrrrmmm... ok .. curious to see if this is helpful

File: lineman/app/components/utilities.scss [hunk 2]
```diff
@@ -261,6 +261,15 @@ label.label-with-details{
   h4 {
     @include fontHelveticaBold;
   }
+  ul{
```
Comment (robguthrie) [inline lineman/app/components/utilities.scss]: arrrh sweet sweet caring for markdown.

**Comment by robguthrie** [comment]
merge whenever you like and I'll deploy

**Comment by rdbartlett** [comment]
cool I might sneak a couple more things in before standup

**Comment by rdbartlett** [comment]
Ok I'm ready for merge now

**Pull request merged**